src/main/java/net/minestom/server/entity/LivingEntity.java

```diff
@@ -384,7 +384,10 @@ public boolean damage(@NotNull Damage damage) {
      * @return true if this entity is immune to the given type of damage
      */
     public boolean isImmune(@NotNull DynamicRegistry.Key<DamageType> type) {
-        return false;
+        if (isInvulnerable() && type == DamageType.OUT_OF_WORLD) {
```

Comment:
I think this logic would be clearer if it was simply:

```
if (type.equals(OUT_OF_WORLD) return false;
return isInvulnerable()
```

Checking isInvulnerable twice seems like it just adds confusion

Reply:
Changed to match.